### PR TITLE
Create CONTRIBUTING.md

### LINKED ISSUE #207: Add CONTRIBUTING.md
Add CONTRIBUTING.md that includes a code of conduct and guidelines for contributing to the project.

### PR DESCRIPTION
**Pull request by hwar042**
Created CONTRIBUTING.md with the basic structure in place

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+### Pull Requests
+
+## Project Resources
+### Adding SVG Images
+### Adding Background Images
+
+## Styleguides
+### Naming Conventions
+Names of project files should:
+* BeWrittenInPascalCase.
```
Comment (franc-hoon) [inline CONTRIBUTING.md]: Names of components should be in PascalCase and end in .jsx. 
Names of function files should be in camelCase and end in .js. 
Names of scss files should also be in camelCase.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+* Be described in three or less words.
+* Include, if necessary, the subgroup name.
+### BEM Naming for CSS
+Mello uses the BEM Naming structure, which aims to keep CSS codebases structured with a consistent naming convention.
+
+*For more information, see https://yandex.com/dev/bem/*
+### File Structure
```
Comment (franc-hoon) [inline CONTRIBUTING.md:66]: Could also mention that scss files should mirror the component file structure

**Review by franc-hoon — CHANGES_REQUESTED** [review]
looking good, just a few additions i think are necessary. lmk if you think otherwise 👍

File: CONTRIBUTING.md (hunk 1 above)
Comment (hwar042) [reply CONTRIBUTING.md]: Appreciate the extra detail, will add this

**Review by brickahax — APPROVED** [review]
Well structured, looking good 👌

File: CONTRIBUTING.md (hunk 2 above)
Comment (hwar042) [reply CONTRIBUTING.md:66]: Added 👍

**Comment by hwar042** [comment]
@M-Stable Changes have been made

**Review by franc-hoon — APPROVED** [review]
Looks Good, thanks for adding that in 💯

**Pull request merged**